.travis.yml

```diff
@@ -51,17 +51,6 @@ jobs:
         on:
           branch: master
 
-# Use travis to cache a Bazel disk-cache.
-# This connects to the docker container via a volume mount (`-v /home/travis/disk-cache:/disk-cache/`).
-# Bazel will store artifacts here due to the `--disk_cache=/disk-cache/` argument.
-cache:
-  npm: true
-
 # Pulling container from https://hub.docker.com/repository/docker/arcproject/travis-build
 before_install:
   - docker pull arcsproject/travis-build:latest
-  - mkdir -p ./disk-cache/
-
-install:
-  - tools/run_in_docker "npm install"
```

Comment:
If you delete this step, the build will automatically run `npm install`, because it is set to the nodejs environment. 

If you don't want to install things in the docker container, at least add a dummy `install:` step.

Reply:
Oooohh, that explains why that was happening then! Thanks